client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,163 @@
+				{ translate(
+					'Following this simple 7-step checklist will help you get your site featured at the top of search engine results for your most important keywords (and help you avoid the most common pitfalls that you absolutely must avoid if you want to rank well).'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'Why email is STILL a must-have marketing channel (hint: it’s simple, free, and effective) (page 88)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'This guide will teach you everything you need to know: from which email marketing tools you should use, to how to collect email addresses and write great subject lines, and more.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate( 'Harness the power of social media to expand your reach (page 65)' ) }
+				</strong>
+				<br></br>
+				{ translate(
+					'Discover the best ways to leverage social media channels like FaceBook, Twitter, and LinkedIn to connect with your audience and turn them into raving fans.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'How to build a simple “traffic magnet” to instantly attract the right audience (page 7)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'It’s so effective you don’t even have to come close to a “perfect” site before you launch… even the ugliest site can drive an endless stream of business if you get this right!'
+				) }
+			</p>
+			<p>
+				{ translate(
+					'If you’ve spent any time looking for marketing help, you know that marketing guides like this often cost $100 or more. But you won’t have to pay anywhere near that amount.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ isLoading ? (
+						<div className="ultimate-traffic-guide__placeholder">
+							<div className="ultimate-traffic-guide__placeholder-animation"></div>
+						</div>
+					) : (
+						translate(
+							'In fact, if you act today you’ll pay just %(cost)s for The Ultimate Traffic Guide.',
+							{
+								args: { cost: formatCurrency( cost, currency, { stripZeros: true } ) },
+							}
+						)
+					) }
+				</strong>
+				<br></br>
+			</p>
+			<p>
+				<Button onClick={ handleSalesPageButtonClick } primary>
+					{ translate( 'Click here to buy the guide now!' ) }
+				</Button>
+			</p>
+		</Card>
+	);
+};
+
+const DownloadPage = ( { translate } ) => {
+	const handleDownloadPageButtonClick = () => {
+		//download
+	};
+
+	return (
+		<Card>
+			<p>
+				{ translate(
+					// eslint-disable-next-line inclusive-language/use-inclusive-words
+					'Congratulations for taking this important step towards mastering the art of online marketing! To download your copy of The Ultimate Traffic Guide, simply click the button below and confirm the download prompt.'
+				) }
+			</p>
+			<p>
+				{ translate(
+					'The Ultimate Traffic GuideThis is a goldmine of traffic tips and how-tos that reveals the exact “Breakthrough Traffic” process we’ve developed over the past decade.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'The Ultimate Traffic Guide is a goldmine of traffic tips and how-tos that reveals the exact “Breakthrough Traffic” process we’ve developed over the past decade.' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/ultimate-traffic-guide)